uwsgi.conf

```diff
@@ -1,9 +1,11 @@
 [uwsgi]
-uid = nginx
+uid = www-data
+gid = www-data
 module = application:application
 
-plugins = /usr/lib/uwsgi/python
 socket = /run/uwsgi.sock
 
+single-interpreter = true
+enable-threads = true
```

Comment:
Short answer: they silence some uwsgi warnings 😄 

Single-interpreter tells uwsgi that we're only running one app, so it can skip some steps when processing requests. Enabled-threads allows us to use Python threads in the apps (without it, uwsgi patches out GIL to prevent context switches within the app)